temp-storage/README.md

```diff
@@ -0,0 +1,19 @@
+# Utilise temporary storage under Functions
+
+This Function shows you how to reach and utilise the temporary storage under the Function layer, mainly for single-invocation jobs
+For example, on each invocation we can create a file based on user data and use it accordingly
+
+IMPORTANT: Do NOT treat this storage as long term storage or for personal data that need to persist.
+The contents get deleted whenever the associated container is brought down, so this function is useful for one time actions
+
+Pre-requisites
+- You need to include the following npm modules: fs, path, os
```

Comment:
These are all built-in modules. So no need to mention them here

Reply:
ok removed from both the function js description and the readme description